### PR TITLE
Manual download of PROJ

### PR DESCRIPTION
**Pull request by SamuilsRulovs**
- [x] Include PROJ4 search in the CMakeLists.txt
- [x] Update SetupHelp.md (Include description of PROJ4 manual download)

Solves a bit of #226 .

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -249,6 +249,9 @@ if(NOT blue_framework_POPULATED)
 	set_target_properties(UpdateLibsAndDLLsForCurrentBuildConfiguration PROPERTIES FOLDER "BlueFramework/Commands")
 endif()
 
+set (CMAKE_PREFIX_PATH "C:/OSGeo4W64/share/cmake/proj4")
+find_package(PROJ4 REQUIRED)
```
Comment (pjanck) [inline CMakeLists.txt]: PROJ4 was superseeded by PROJ. Please update to the newest version as described [here](https://proj.org/development/cmake.html).

File: Documentation/markdown/SetupHelp.md [hunk 2]
```diff
@@ -50,7 +50,24 @@ These steps need to be completed before you can proceed with compiling OIP.
 	* Add this path to the environment variables. (Create new environment variable called `Boost_INCLUDE_DIR`. This variable should point to the binary folder, where **Boost 1_65_1** is staged (e.g. `C:\thirdparty\vs2017\x64\boost_1_65_1`).
 
 5. **Anaconda 2** (version with Python 2.7) - find [here](https://repo.anaconda.com/archive/Anaconda2-2019.10-Windows-x86_64.exe). 
+6. **PROJ 4:**
+
+	*	Download OSGeo4W64 Installer (find [here](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86_64.exe)).
+	*	In Installer select *Advanced Install*
+	*	Choose to install from Internet
+	*	Select directory, where PROJ 4 will be installed (`C:\OSGeo4W64` should be default option).
```
Comment (pjanck) [inline Documentation/markdown/SetupHelp.md]:
```suggestion
	*	Select directory, where PROJ 4 will be installed
	NOTE: CMake expects it at `C:\OSGeo4W64`, which should be the default option. If you deviate from this, please set the `PROJ_INSTALL_DIR` variable correspondingly.
```

Führe eine neue CMake variable ein: `PROJ_INSTALL_DIR`, die per default auf `C:\OS4Geo4W64` zeigt. Die `CMAKE_PREFIX_PATH` sollte dann mit der Variable besetzt sein.

File: Documentation/markdown/SetupHelp.md [hunk 3]
```diff
@@ -50,7 +50,24 @@ These steps need to be completed before you can proceed with compiling OIP.
...
+	*	Select local package directory (e.g. `C:\local`)
```
Comment (pjanck) [inline Documentation/markdown/SetupHelp.md]: The local package directory could be in `OSGeo4W64\local`.

File: Documentation/markdown/SetupHelp.md [hunk 4]
```diff
@@ -50,7 +50,24 @@ These steps need to be completed before you can proceed with compiling OIP.
...
+	*	Select `https://osgeo4w-oslandia.com/` as a download site.
+	*	Select components to install:
+		
+		* Open *Commandline_Utilities*
+		* Find *proj* package 
+		* Click on skip until PROJ version 6.* show up.
```
Comment (pjanck) [inline Documentation/markdown/SetupHelp.md:68]: Why can't we install versions 7.+?

**Review by pjanck — CHANGES_REQUESTED** [review]
I thought the official version is 7.2.0? It would be great to have the newest developments.

File: CMakeLists.txt (hunk 1 above)
Comment (SamuilsRulovs) [reply CMakeLists.txt]: Resolved: efdc4b3

File: Documentation/markdown/SetupHelp.md (hunk 3 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md]: Resolved: 21b0946

File: Documentation/markdown/SetupHelp.md (hunk 4 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md:68]: Resolved: efdc4b3

File: CMakeLists.txt [hunk 5]
```diff
@@ -249,8 +253,12 @@ if(NOT blue_framework_POPULATED)
 	set_target_properties(UpdateLibsAndDLLsForCurrentBuildConfiguration PROPERTIES FOLDER "BlueFramework/Commands")
 endif()
 
-set (CMAKE_PREFIX_PATH "C:/OSGeo4W64/share/cmake/proj4")
-find_package(PROJ4 REQUIRED)
+
+#set (PROJ_INSTALL_DIR "C:/OSGeo4W64/apps/proj-dev/share/cmake/proj4/proj")
+#set (CMAKE_PREFIX_PATH PROJ_INSTALL_DIR)
+
```
Comment (pjanck) [inline CMakeLists.txt]: Remove the commented out stuff that is a remnant of a failed attempt. We try to keep the code at least at the same level of chaos as it was before and not add to it :)

File: Documentation/markdown/SetupHelp.md [hunk 6]
```diff
@@ -50,13 +50,16 @@ These steps need to be completed before you can proceed with compiling OIP.
...
+6. **PROJ :**
 
 	*	Download OSGeo4W64 Installer (find [here](http://download.osgeo.org/osgeo4w/osgeo4w-setup-x86_64.exe)).
 	*	In Installer select *Advanced Install*
 	*	Choose to install from Internet
-	*	Select directory, where PROJ 4 will be installed (`C:\OSGeo4W64` should be default option).
-	*	Select local package directory (e.g. `C:\local`)
+	*	Select directory, where PROJ will be installed.
+
+	NOTE: CMake expects it at `C:\OSGeo4W64`, which should be the default option. If you deviate from this, please set the `PROJ_INSTALL_DIR` variable correspondingly.
```
Comment (pjanck) [inline Documentation/markdown/SetupHelp.md]: This is not true anymore. Please correct with the correct CMake variable name.

**Review by pjanck — CHANGES_REQUESTED** [review]
Small tweaks to documentation.

File: CMakeLists.txt (hunk 5 above)
Comment (SamuilsRulovs) [reply CMakeLists.txt]: Resolved: 213c48b

File: Documentation/markdown/SetupHelp.md (hunk 6 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md]: Resolved: 213c48b

File: Documentation/markdown/SetupHelp.md (hunk 2 above)
Comment (SamuilsRulovs) [reply Documentation/markdown/SetupHelp.md]: Resolved: 213c48b

**Review by pjanck — APPROVED** [review]

**Pull request merged**